account_invoice_pricelist_sale/tests/test_module.py

```diff
@@ -32,14 +29,30 @@ def test_main(self):
                             "price_unit": self.product.list_price,
                             "qty_delivered": 5,
                         },
-                    ),
+                    )
                 ],
             }
         )
+
+    def test_invoice_with_pricelist(self):
+        """Test invoice creation with pricelist"""
+        order = self._create_sale_order(
+            pricelist=self.env["product.pricelist"].create({"name": "Demo Pricelist"})
+        )
```

Comment:
```suggestion
pricelist = self.env["product.pricelist"].create({"name": "Demo Pricelist"})
order = self._create_sale_order(pricelist=pricelist)
```
Enough with those 3 lines one-liners :pray: